docs/integrations-guides/rollup-guides/op-stack/README.md

```diff
@@ -22,7 +22,10 @@ First check out the version of the EigenDA proxy corresponding to the version of
 |------------------|-----------------------|
 | [v1.7.6](https://github.com/ethereum-optimism/optimism/releases/tag/v1.7.6)           | [v1.0.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.0.0)                |
 | [v1.7.7](https://github.com/ethereum-optimism/optimism/releases/tag/v1.7.7)           | [v1.2.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.2.0)                |
-| [v1.9.0](https://github.com/ethereum-optimism/optimism/releases/tag/v1.9.0)           | [v1.4.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.4.0)
+| [v1.9.0](https://github.com/ethereum-optimism/optimism/releases/tag/v1.9.0)           | [v1.4.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.4.0)                |
+| [v1.9.3](https://github.com/ethereum-optimism/optimism/releases/tag/v1.9.3)           | TODO(@bxue): make new release                |
+
+The latest v1.9.2 op-stack release does not contain our most recent concurrent batch submission [PR](https://github.com/ethereum-optimism/optimism/pull/11698), hence high-throughput chains will need to use the latest op-challenger v1.1.0 release until the op-stack v1.9.3 release.
```

Comment:
it is no longer latest. Are we comfortable with 1.9.3 now?

Reply:
Yes. Updated to [af4deb1](https://github.com/Layr-Labs/eigenda-docs/pull/20/commits/af4deb1def3750a96f35ad9f176e3023b3ba04c6)